tests/db_access/driver.cpp

```diff
@@ -23,6 +24,19 @@ go_bandit([](){
             DBDriver driver{};
             driver.connect(s.host.c_str(), s.user.c_str(), s.password.c_str(), nullptr, s.port, nullptr);
         });
+
+        it("can make hex string from string types", [&]() {
+            DBDriver driver{};
+            std::string expected{"666F6F62617200"};
+
+            AssertThat(driver.makeHexString("foobar"), Equals(expected));
+            AssertThat(driver.makeHexString("foobar"s), Equals(expected));
+
+            std::string foobar{"foobar"};
+            AssertThat(driver.makeHexString(foobar), Equals(expected));
```

Comment:
Isn't this test already on line 33? :)

Reply:
Nope, assert on line 33 calls method with rvalue, this assert calls method with lvalue.

Reply:
Ok, however after the last change *makeHexString()* takes const lvalue, thus it does the same thing and so this is redundant test :) .

Reply:
ACK